javascript/javascript3/week3/shoppingCart/script.js

```diff
@@ -0,0 +1,80 @@
+//Shopping cart using Classes
+const searchInput = document.getElementById("searchInput");
+
+class Product {
+    constructor(name, price) {
+        this.name = name;
+        this.price = price;
+    }
+    convertToCurrency(currency) {
+        const rate = { EUR: 0.13, USD: 0.14, INR: 10.94 };
+        return (this.price * rate[currency]).toFixed(2);
+    }
+}
+
+class ShoppingCart {
+    constructor() {
+        this.products = [];
+    }
+
+    addProduct(product) {
+        this.products.push(product);
+    }
+    removeProduct(product) {
+        const index = this.products.indexOf(product)
+        this.products.splice(index, 1)
+    }
+
+    searchProduct(productName) {
+        const searchProduct = this.products.filter(item => item.name.toLowerCase().includes(productName.toLowerCase()));
+        return searchProduct;
+    }
+
+    getTotal() {
+        const totalPrice = this.products.map(product => product.price);
+        let total = 0;
+        for (let i = 0; i < totalPrice.length; i++) {
+            total += totalPrice[i];
+        }
+        return total;
+    }
+    renderProducts() {
+        const shoppingCart = document.getElementById("shoppingCart");
+        this.products.forEach(pd => {
+            const ul = document.createElement("ul");
+            ul.innerHTML = ` Product Name: ${pd.name}`
+            shoppingCart.appendChild(ul);
+            const li = document.createElement("li");
+            li.innerHTML = `Price: ${pd.price}`
+            ul.appendChild(li)
+        })
+        const userDetail = document.createElement("p");
+        console.log(this.getUser(1))
```

Comment:
delete console.log in final code :)

Reply:
Yes I always check in console and then forget to delete it 😃  but will do that